requirements/edx-sandbox/py35.txt

```diff
@@ -51,7 +51,7 @@ networkx==2.2
     # via
     #   -c requirements/edx-sandbox/../constraints.txt
     #   -r requirements/edx-sandbox/py35.in
-nltk==3.6.4
```

Comment:
Is this codejail version downgrade required?

Reply:
This release has been yanked. 

https://pypi.org/project/nltk/3.6.4/

Reply:
Oooh, got it. :-/